it/test/uk/gov/hmrc/incometaxpenaltiesappealsfrontend/repositories/UserAnswersRepositoryISpec.scala

```diff
@@ -0,0 +1,107 @@
+/*
+ * Copyright 2025 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.incometaxpenaltiesappealsfrontend.repositories
+
+import play.api.libs.json.Json
+import play.api.test.Helpers._
+import uk.gov.hmrc.incometaxpenaltiesappealsfrontend.models.session.UserAnswers
+import uk.gov.hmrc.incometaxpenaltiesappealsfrontend.utils.ComponentSpecHelper
+
+class UserAnswersRepositoryISpec extends ComponentSpecHelper {
+
+  lazy val repository: UserAnswersRepository = injector.instanceOf[UserAnswersRepository]
+
+  override def beforeAll(): Unit = {
+    super.beforeAll()
+  }
+
+  override def beforeEach(): Unit = {
+    super.beforeEach()
+  }
+
+  override def afterEach(): Unit = {
+    super.afterEach()
+  }
+
+  override def afterAll(): Unit = {
+    super.afterAll()
+  }
```

Comment:
These aren't needed, because we're not adding anything extra to the `before` or `after` functions. 
```suggestion
```